R/mapView.R

```diff
@@ -881,65 +914,65 @@ setMethod('mapView', signature(x = 'sfc'),
 #'
 #' @describeIn mapView \code{\link{character}}
 setMethod('mapView', signature(x = 'character'),
```

Comment:
@lbusett why did you change things for the character method? Are you sure this is intended?

Reply:
Not at all - I only changed the "stars" method. I thought that the other highlighted changes were due to re-indentation, but maybe I changed something accidentally or while rebasing my local branch with remotes. 

Any change besides the stars method are definitely accidental... sorry - sometimes git has the better of me ;-) .

Reply:
No problem, but can you revert those changes?

Reply:
arg... I'll try... easiest way is to do a fresh clone...  ;-)

Reply:
can we delete this PR ? I'll try to do a clean one.

Reply:
sure, the cleaner the better, my PR skills are also limited

Reply:
ok. I tried a new one. Seems to be better.